zulipterminal/ui_tools/views.py

```diff
@@ -1929,6 +1942,57 @@ def keypress(self, size: urwid_Size, key: str) -> str:
         return super().keypress(size, key)
 
 
+class ReadReceiptView(PopUpView):
+    def __init__(
+        self,
+        controller: Any,
+        message: Message,
+        topic_links: "OrderedDict[str, Tuple[str, int, bool]]",
+        message_links: "OrderedDict[str, Tuple[str, int, bool]]",
+        time_mentions: List[Tuple[str, str]],
+        title: str,
+    ) -> None:
+        self.controller = controller
+        self.message = message
+        self.topic_links = topic_links
+        self.message_links = message_links
+        self.time_mentions = time_mentions
+        max_cols, max_rows = controller.maximum_popup_dimensions()
+
+        # Get rendered message header and footer
+        msg_box = MessageBox(message, controller.model, None)
+
+        # Get read receipt user ids list
+        response = controller.model.fetch_read_receipts(message["id"])
+        if response is None:
+            return
```

Comment:
I wrote the ReadReceiptView class based on the `FullRawMsgView` because it seems the most similar. They've handled it exactly like that.

Reply:
Yes, that does seem the case. You might try testing to see what happens when you disconnect from the internet and then press `i`+`r` (or `l` if it's still set that way). I'd think it would be good to show the user something in this case, and similarly apply a bug fix for the Raw message view.

Reply:
`i`+`r` wouldn't work because pressing `i` closes the info popup immediately. I've changed it to `ctrl`+`r`. So final keypress sequence would be `i` to open the message information popup and then `ctrl`+`r`.


Re the uninitialised object, I'm thinking of showing an error message in the response instead of the list of people who've read. If that looks okay will open a separate PR for `FullRawMsgView`